pkg/jx/cmd/step_create_task.go

```diff
@@ -490,7 +490,7 @@ func (o *StepCreateTaskOptions) CreateTaskForBuildPack(languageName string, pipe
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:   name,
-			Labels: util.MergeMaps(o.labels, map[string]string{syntax.LabelStageName: syntax.DefaultStageNameForBuildPack}),
+			Labels: util.MergeMaps(o.labels, map[string]string{syntax.LabelStageName: syntax.MangleToRfc1035Label(syntax.DefaultStageNameForBuildPack, "")}),
```

Comment:
Would still be good to get rid of this function in favor of `kube.toValidName*`, though I'm not sure if the import cycles are still a problem or not.

Reply:
Lemme check.

Reply:
Bleh, still a problem. I'll deal with that later.